app/controllers/evaluators_controller.rb

```diff
@@ -21,14 +22,30 @@ def create
 
   def destroy
     result = evaluator_service.remove_evaluator(params[:evaluator_type], params[:id])
-    render_json_response(result)
+
+    if result[:success]
+      flash[:notice] = result[:message]
+      render json: { success: true, message: result[:message] }
+    else
+      render json: { success: false, message: result[:message] }, status: :unprocessable_entity
+    end
```

Comment:
again just inlining this so it's easier to see what is rendered by the `destroy` action